src/Fire.java

```diff
@@ -0,0 +1,39 @@
+public class Fire extends SuperPokemon {
+
```

Comment:
Voor de duidelijkheid zou ik deze klasse liever FirePokemon noemen. Wees in het maken van klassen en variable zo specifiek mogelijk. Liever een lange duidelijke naam dan een korte naam die meerdere interpetaties kan hebben